.gitignore

```diff
@@ -16,3 +16,6 @@
 *.iml
 .idea
 .vscode
+
+/ampd/state.json
```

Comment:
Not important, but technically this can be done in `ampd/.gitignore`. A repo can have multiple gitignore files. I'll leave this up to you.